sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/WebSocketsProxyConnectionHandler.java

```diff
@@ -39,10 +39,23 @@
 public class WebSocketsProxyConnectionHandler extends WebSocketsConnectionHandler {
```

Comment:
Is it possible to test that the fields we expect to be set are correct? (ie. unit test)

Reply:
Make sense, I've added four tests to avoid regression.